src/app.ts

```diff
@@ -146,4 +189,32 @@ export default (
       method: 'deleteSecret',
     });
   });
-};
+
+  app.on("issues.opened", async(context: any) => {
+    let issueComment = context.issue({
+           body: "Thanks for submitting onboarding request!",
+         });
+    
+    let data;
+    try {
+      data = await parse(context);
+    } catch {
+      app.log.info('Issue was not created using Issue form template (the YAML ones)');
+    }
+
+    const body:string = context.payload.issue["body"];
+    if (body.includes("### Target cluster")){ //Used to check if it is a onboarding request
+
+      if (data["quota"][0] == "custom"){
+        data["quota"] = data["custom-quota"];
+      };
+  
+      const payload = JSON.stringify(data); //format data to send to task
+  
+      createTaskRun('robozome', payload);
```

Comment:
Shouldn't the second parameter here be `context` (and `payload` the third parameter).